src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -165,4 +175,28 @@ public void validateEmail(String email) throws SubscriptionValidationException {
             throw new SubscriptionValidationException("Wrong email address: " + email);
         }
     }
+
+    public void validateWithSchema(Subscription subscription) throws SubscriptionValidationException {
+        LOGGER.debug("Validation of subscription " + subscription.getSubscriptionName() + " Started.");
+
+        ObjectMapper mapper = new ObjectMapper();
+        String subscriptionJson = null;
```

Comment:
Since it becomes 2 try/catch in one function and they do 2 different things, it is recommended that a function serve a single purpose, it could look better if the firsat try/catch was in its own function.